http_logs/challenges/default.json

```diff
@@ -91,6 +91,35 @@
           "warmup-iterations": 200,
           "iterations": 100,
           "target-throughput": 0.5
+        },
+        {
+          "name": "force-merge-1-seg",
+          "operation": {
+            "operation-type": "force-merge",
+            "max-num-segments": 1
```

Comment:
Can you please also add `request-timeout`and set it to a value that is high enough (I guess `7200` (2 hours) should suffice) to ensure we'll wait long enough for this operation to finish? Otherwise we might run queries while the force merge is still ongoing in the background.

Reply:
@danielmitterdorfer Thanks for the feedback. Addressed in 77faa8b